docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -59,10 +64,16 @@ def transform_fn(model, request_body, input_content_type, output_content_type="a
 
     pred = model.predict(data)
     pred_proba = model.predict_proba(data)
-    prediction = pd.concat([pred, pred_proba], axis=1).values
+    prediction = pd.concat([pred, pred_proba], axis=1)
 
-    return json.dumps(prediction.tolist()), output_content_type
+    return prediction.to_json(), output_content_type
 ```
+For inference with other types of AutoGluon Predictors, i.e. TextPredictor, the inference script you provided will be quite similar to the one above.
+Mostly, you just need to replace `TabularPredictor` to be `TextPredictor` for example.
```

Comment:
```
globals()["column_names"] = model.feature_metadata_in.get_features()
model.persist_models()
```
I guess this code doesn't work for `MultiModalPredictor`. Could we add if check in `model_fn`?

Reply:
Yes, but I did add a comment there to remind the user:
```
# Get original columns for header validation later
# Different predictors might have different ways to retrieve this info
# Free to remove this check if you know your test data is always valid.
```